### PR TITLE
k3s: add script to setup a k3s cluster with s3gw

### PR DESCRIPTION
**Pull request by jecluis**
Sets up a single-node k3s cluster with s3gw consuming a longhorn pvc.

Signed-off-by: Joao Eduardo Luis \<joao@suse.com>

### CODE REVIEW

**Review by votdev — APPROVED** [review]

File: k3s/longhorn-pvc.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: longhorn-s3gw-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
```
Comment (giubacc) [inline k3s/longhorn-pvc.yaml:7]: The mode `ReadWriteOnce` means that if the pod bound to the pvc restarts it will get a brand new filesystem?

Comment (giubacc) [inline k3s/longhorn-pvc.yaml:7]: The mode `ReadWriteOnce` means that if the pod bound to the pvc restarts it will get a brand new filesystem?

Comment (votdev) [reply k3s/longhorn-pvc.yaml:7]: No, see https://kubernetes.io/blog/2021/09/13/read-write-once-pod-access-mode-alpha/#how-is-this-different-than-the-readwriteonce-access-mode

**Comment by jecluis** [comment]
merged via #23

**Pull request closed without merging**